opensrp-chw-core/src/test/java/org/smartregister/chw/core/activity/CoreAncRegisterActivityTest.java

```diff
@@ -66,4 +97,42 @@ public void testGetFamilyBaseEntityId() {
         Assert.assertEquals(activity.getFamilyName(), "familyName");
         Assert.assertEquals(activity.getPhoneNumber(), "phone_number");
     }
+
+    @Test
+    public void testOnResumption() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`